### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by heorhiikalaichev** [mingyu6yang/striped-wrinet#2]
Hi!

I'm wondering how you achieved so good results!
Are you going to add here loss function and training script?
